server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -262,8 +263,16 @@ mkCompExpInp colTy =
       PGGeometry -> True
       _          -> False
 
-    geomOpToInpVal (op, desc) =
+    isGeographyTy = case colTy of
+      PGGeography -> True
+      _          -> False
+
+    geometryOpToInpVal (op, desc) =
       InpValInfo (Just desc) op Nothing $ G.toGT $ mkScalarTy PGGeometry
+
+    geographyOpToInpVal (op, desc) =
```

Comment:
`geographyOpToInpVal` and `geometryOpToInpVal` has similar code. Merge them.

Reply:
Done